dora/core/common/src/main/java/alluxio/wire/WorkerIdentity.java

```diff
@@ -28,15 +53,19 @@
  * representation and an instance of this class.
  */
 @Immutable
-public final class WorkerIdentity {
+@JsonAdapter(WorkerIdentity.GsonSerde.class)
+@JsonDeserialize(converter = WorkerIdentity.JacksonDeserializeConverter.class)
+@JsonSerialize(using = WorkerIdentity.JacksonSerializer.class)
```

Comment:
why do we need two json serializer impl for jackson and gson?

Reply:
I think maybe gson is much easier to serialize and deserialize the object manually.

Reply:
`WorkerInfo` needs to be de/serialized by Jackson, and `WorkerIdentity` is embedded in `WorkerInfo`. See `alluxio.wire.WorkerInfoTest#json`